frontend/Auth/FinishResetPassword.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from "react";
+import { Button, ButtonKind } from "../Styleguide/Button";
+import { FormField } from "../Styleguide/FormField";
+import { FormFieldLabel } from "../Styleguide/FormFieldLabel";
+import { Input } from "../Styleguide/Input";
+import { RouterProps } from "react-router";
+import { useMutation, useQuery } from "react-query";
+import { flaxFetch } from "../Utils/flaxFetch";
+import { unary } from "lodash-es";
+
+export function FinishResetPassword(props: RouterProps) {
+  const [finishResetPasswordFormData, setFinishResetPasswordFormData] =
+    useState<FinishResetPasswordFormData>({
+      password: "",
+    });
+
+  const paramString = props.history.location.search;
+  let searchParams = new URLSearchParams(paramString);
+  const token = searchParams.get("jwt") || "MISSING TOKEN";
+
+  const queryFunctionhelper = (queryKey) => {
```

Comment:
It's more common just to inline this into where you're calling `useQuery()`. There's nothing wrong with making it a separate variable, though. If you keep it as a separate variable, fix the casing to be camel case with `queryFunctionHelper`

Reply:
Absolutely!  Good catch.  I fixed it in a future PR but will fix it here too.